xcube/api/gen/rbins/iproc.py

```diff
@@ -52,14 +52,50 @@ def get_reprojection_info(self, dataset: xr.Dataset) -> ReprojectionInfo:
                                 xy_crs=CRS_WKT_EPSG_4326,
                                 xy_gcp_step=1)
 
-    def get_time_range(self, dataset: xr.Dataset) -> Tuple[float, float]:
+    def get_time_range_v1(self, dataset: xr.Dataset) -> Tuple[float, float]:
         date = dataset.attrs.get('DATE')
         if date is None:
             raise ValueError('illegal L2 input: missing DATE attribute')
         time = dataset.attrs.get('TIME', '1200')
         days_since_1970 = get_time_in_days_since_1970(date + time)
         return days_since_1970, days_since_1970
 
+    def get_time_range(self, dataset: xr.Dataset) -> Tuple[float, float]:
```

Comment:
This looke very much like the implementation in `DefaultInputProcessor.get_time_range()`.
Try deriving your class from `DefaultInputProcessor` so we don't duplicate code and only need a single test:

```
class RbinsSeviriHighrocSceneInputProcessor(DefaultInputProcessor):
    ...
```